woocommerce/woocommerce-gutenberg-products-block
# Prepare the Mini Cart block for WC Core release

Fixes #6200 

This PR prepares the Mini Cart block to be merged in the WC Core release.




### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Run `npm run build`.
2. Replace the content of`.blocks.ini` with the following config:
```
woocommerce_blocks_phase = 1
woocommerce_blocks_env = production
```
3. See the Mini Cart block and Mini Cart blocks content work as expected.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
